webapp/app/js/controllers/main.js

```diff
@@ -48,10 +48,17 @@ treeherder.controller('MainCtrl', [
             }
         };
 
-        $scope.processKeyboardInput = function(ev){
+        // Disable single key shortcuts in specified shortcut events
+        $scope.allowKeys = function() {
+            Mousetrap.unbind(['i', 'j', 'n', 'k', 'p', 'space', 'u', 'r', 'c']);
```

Comment:
Wondering it might be useful to annotate the unbind, with an adjacent comment. Since we will exceed that 80 char line length when we add `left` and `right` arrow shortcuts shortly. eg.
```
Mousetrap.unbind([
  'i',    // Toggle display in-progress jobs (pending/running)
  'j',    // Select next unclassified failure
  'n',    // Select next unclassified failure
```

If not now, perhaps an idea for later.